orca-front50/src/main/groovy/com/netflix/spinnaker/orca/front50/spring/DependentPipelineExecutionListener.groovy

```diff
@@ -51,8 +74,17 @@ class DependentPipelineExecutionListener implements ExecutionListener {
     }
 
     def status = convertStatus(execution)
+    def allPipelines = front50Service.getAllPipelines()
+    if (pipelinePreprocessors) {
+      // Resolve templated pipelines if enabled.
+      allPipelines = allPipelines.findAll { it.type == 'templatedPipeline'}
+      .collect { pipeline ->
+        V2Util.planPipeline(contextParameterProcessor, pipelinePreprocessors, pipeline)
```

Comment:
Do we need to gate this to only get called on v2 templates, or is it a no-op for v1?

Reply:
We should no-op for v1 to preserve behavior.